Bottom: 9e07f55d70d0b2a5b8d566e561dd1f94cbc1d4b1
Top:    e2643b3718b105a652eb7e1ea6af32605792adb3
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-04-21 00:30:36 +0100

Refresh of work-in-progress-on-subcomposi

---

diff --git a/src/draw.c b/src/draw.c
index 7df5880..9f02cd6 100644
--- a/src/draw.c
+++ b/src/draw.c
@@ -861,7 +861,7 @@ DrawLayerCommon (LayerTypePtr Layer, const BoxType * screen, bool clear_pins)
 
   /* HACK: Subcomposite polygons separately from other layer primitives */
   /* Reset the compositing */
-  gui->set_layer (0, GetGroupOfLayer (Layer), 0);
+  gui->set_layer (0, GetLayerGroupNumberByPointer (Layer), 0);
 
   if (clear_pins && TEST_FLAG (CHECKPLANESFLAG, PCB))
     return;
